src/main/java/trade/wayruha/mexc/dto/Order.java

```diff
@@ -4,64 +4,117 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
-import trade.wayruha.mexc.enums.OrderSide;
-import trade.wayruha.mexc.enums.OrderStatus;
-import trade.wayruha.mexc.enums.OrderType;
-import trade.wayruha.mexc.enums.TimeInForce;
+import trade.wayruha.mexc.enums.*;
 
 import java.math.BigDecimal;
 
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class Order{
+public class Order {
+
     private String symbol;
+
     @JsonAlias({"i"})
     private String orderId;
-    private long orderListId;
+
+    private Long orderListId;
+
     @JsonAlias({"c"})
     private String clientOrderId;
-    @JsonAlias({"p"})
-    private BigDecimal price;
-    @JsonAlias({"ap"})
-    private BigDecimal avgPrice;
-    @JsonAlias({"v"})
-    private BigDecimal origQty;
-    private BigDecimal executedQty;
-    @JsonAlias({"A"})
-    private BigDecimal remainAmount;
-    @JsonAlias({"cv"})
-    private BigDecimal cumulativeQuoteQty;
-    @JsonAlias({"ca"})
-    private BigDecimal cumulativeAmount;
-    @JsonAlias({"a"})
-    private BigDecimal origQuoteOrderQty;
-    private OrderStatus status;
-    private TimeInForce timeInForce;
-    private OrderType type;
-    private OrderSide side;
-    private BigDecimal stopPrice;
-    private BigDecimal icebergQty;
-
-    @JsonAlias({"O", "createTime"})
-    private long time;
-    private long updateTime;
-    private boolean isWorking;
-    @JsonAlias({"m"})
-    private boolean isMaker;
 
+    private OrderType type;
     @JsonProperty("o")
     public void setOrderTypeByAlias(int aliasNumber) {
         this.type = OrderType.findByAlias(aliasNumber);
     }
 
```

Comment:
1.mixing fields & methods declaration does not follow recommended code style and makes it hard to read & navigate in the code. 
2. Do we need @JsonProperty on the setter? Is there a reason why we don't control de-serialization in the OrderSide (e.g., with @JsonValue)? This way, we don't need the custom setter but leave the enum-value resolving logic in the enum itself

Reply:
WebSocket and REST send different type of values (WS send numbers, REST send string). This kind of using JsonProperty resolve this in simply way, creation of custom deserializer was the longest.